lang/en/facetoface.php

```diff
@@ -252,6 +252,10 @@
 $string['moreinfo'] = 'More info';
 $string['multidate'] = '(multi-date)';
 $string['multiple'] = 'Multiple';
+$string['multiplesignupmethod'] = 'Multiple signup method';
+$string['multiplesignupmethod_help'] = 'When set to "single" each session within the activity will have an individual sign up link. When set to "multiple" only one link wil be displayed for the entire activity, any student who registers using this single link will be registered for all sessions within the activity.';
```

Comment:
After reading this I still not really understand the difference between "Per session" vs "Per activity".

Reply:
I've added a [fixup commit](https://github.com/catalyst/moodle-mod_facetoface/pull/93/commits/f299461ce032dd990444712931011791b82b25b9) that attempts to make this feature clearer. It's a bit of a challenge since the plugin isn't really supposed to work this way and we are kind of shoe-horning functionality in :s

Reply:
Looks like there is the same help text for "Signup type" and "Multiple signup method" setting. That's why I was complaining about.

![image](https://user-images.githubusercontent.com/4626431/196096376-082d7cb5-05d1-4a4d-bcfe-fd95a3a6fb16.png)

Reply:
Ah I see what happened. Actually they are different, but I made a typo by writing "single" and "multiple" instead of "Per session" and "Per activity". I've fixed that now.